app/src/main/java/org/fossasia/pslab/fragment/HelpAndFeedbackFragment.java

```diff
@@ -0,0 +1,48 @@
+package org.fossasia.pslab.fragment;
+
+
+import android.os.Bundle;
+import android.support.design.widget.AppBarLayout;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+
+import org.fossasia.pslab.R;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * Created by Harsh on 16-05-2018.
+ */
+
+public class HelpAndFeedbackFragment extends Fragment {
+
+    public static HelpAndFeedbackFragment newInstance() {
+        return new HelpAndFeedbackFragment();
+    }
+
+    @BindView(R.id.appBarAnim)
+    AppBarLayout appBarLayout;
```

Comment:
In Java we normally define variables at the beginning of class. line `27`, `28` should come to `22`. Please make the changes in @abhinavraj23 's similar implementation of About Us fragment as well. It seems like you've just copy pasted his code.